api/conf/settings/spectacular.py

```diff
@@ -50,19 +50,17 @@
             "name": "auth",
             "description": dedent(
                 """
-                The API has rate-limiting and throttling in place to prevent
```

Comment:
I chose not to list out all the details of what the limits are for throttling or privileged pagination. The throttle limits were wrong, sort of, in that those are the limits for _some_ views, but thumbnails and some others have different limits. I don't think we actually need to publish the configured limits explicitly, and doing it like this (when we know they are configured via environment variables elsewhere anyway) inevitably leads to inaccurate information and drifting documentation.

Instead, I've changed this to describe the general idea of Openverse's API having intentional limits to requesters, how we communicate those limits (kept from the existing docs), and letting the reader know they can request higher limits if they wish.